chat-plugins/dice-commands.js

```diff
@@ -1,185 +1,164 @@
-/* Dice commands chat-plugin
- * by jd
-*/
-var tax = 0.10; // 10%
+/*Dice Game plugin by SilverTactic (Silveee)*/
 
-function handleWinnings (bet) {
-	return bet - Math.round(bet * tax);
+'use strict';
+
+const GC = Rooms('gamechamber');
```

Comment:
But it exists on gold, so it doesn't matter lol
(there're several files in chat-plugins that would otherwise crash if certain json files weren't present btw)